api/v01/entities/vrp_input.rb

```diff
@@ -276,10 +276,10 @@ module VrpMisc
   params :vrp_request_route do
     optional(:vehicle_id, type: String, desc: 'Vehicle linked to the current described route')
     optional(:indice, type: Integer, documentation: { hidden: true }, desc: '[ DEPRECATED : use day_index instead ]')
-    optional(:index, type: Integer, desc: 'Index of the route. Must be provided if first_solution_strategy is \'periodic\'.')
```

Comment:
Is this parameter already in use ? If so, we should ensure the retro-compatibility

Reply:
It is not since this param is not connected to anything at the moment. That's why I made the modif as such plus this is the fix that was taken back by a bad rebase.

This is due to the API logic split refacto which you had to rebase multiple times :( In between one of the rebases, this param is corrected both on the API and in the model; however, since your refacto  was splitting the API in multiple files vrp_input.rb etc.. and you haven't do the copy/paste from scratch in each refacto, the model part of the correction has stayed but the API part of the correction is lost.